src/OpenTelemetry/Metrics/Exemplar/Exemplar.cs

```diff
@@ -0,0 +1,54 @@
+// <copyright file="Exemplar.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+
+namespace OpenTelemetry.Metrics
+{
+    /// <summary>
+    /// Represents an Exemplar data.
+    /// </summary>
+    public struct Exemplar
+    {
+        /// <summary>
+        /// Gets the timestamp.
```

Comment:
Could we make it TimestampUtc to avoid confusion? Maybe we have made this mistake too many places already 😄

Reply:
sticking with timestamp itself to be consistent with MetricPoint start/end times. Adding document to make it clear its utc.